activesupport/lib/active_support/core_ext/object/to_query.rb

```diff
@@ -70,14 +78,14 @@ class Hash
   #   {name: 'David', nationality: 'Danish'}.to_query('user')
   #   # => "user%5Bname%5D=David&user%5Bnationality%5D=Danish"
   #
+  # If `escape: false` is provided values are not escaped.
+  #
   # The string pairs "key=value" that conform the query string
   # are sorted lexicographically in ascending order.
-  #
-  # This method is also aliased as +to_param+.
```

Comment:
This is redundant because RDoc already shows such info: "Also aliased as: to_param"